day-2/kavya-challenge-2/src/App.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+
+var index = 0;
+var quotes = ["Believe in your fucking self.", "Stay up all fucking night.", "Use fucking spell check.", 
+        "Don't fucking procrastinate.","Get over your fucking self.", "Keep fucking learning.", "Fucking network.", 
+        "Asking for fucking help.", "Do your fucking research."]
+var len = quotes.length;
+class App extends Component {
+  
+  
+  constructor(props) {
+    super(props);
+    this.state = {currentIndex: index};
```

Comment:
`currentIndex` implies there is a `non-currentIndex` in use. Maybe something like `quoteIndex` would be more precise